src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,19 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private Color[] colors = Color.values();
+    private Random random = new Random();
+    private String randomColor;
+
+    private void setRandomColor() {
+        int index = random.nextInt(colors.length);
+        this.randomColor = colors[index].toString();
+    }
+
     public String getRandomColor() {
-        return null;
+        setRandomColor();
+        return randomColor;
```

Comment:
1) There is no need to use class variable for single usage
2) String representation of ENUM should be obtained with name() method
```
public String getRandomColor() {
    int index = random.nextInt(colors.length);
    return colors[index].name();